modules/op-release-notes-1-6.adoc

```diff
@@ -244,4 +244,23 @@ time="2021-11-04T13:05:26Z" level=error msg="exit status 127"
 * Before this update, editing the specification of a running pipeline might prevent the pipeline run from stopping when it was complete. This update fixes the issue by fetching the definition only once and then using the specification stored in the status for verification. This change reduces the probability of a race condition when a `PipelineRun` or a `TaskRun` refers to a `Pipeline` or `Task` that changes while it is running.
 //  (link:https://issues.redhat.com/browse/SRVKP-718[SRVKP-718])
 
-* `When` expression values can now have array parameter references, such as: `values: [$(params.arrayParam[*])]`.
\ No newline at end of file
+* `When` expression values can now have array parameter references, such as: `values: [$(params.arrayParam[*])]`.
+
+
+
+[id="release-notes-1-6-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.6.1
+
+The following improvements are made in {pipelines-title} 1.6.1:
+
+* The `SSL_CERT_DIR` environment variable (`/tekton-custom-certs`) set by {pipelines-title} will not override the following default system directories with certificate files:
+** `/etc/pki/tls/certs`
+** `/etc/ssl/certs`
+** `/system/etc/security/cacerts`
+// https://issues.redhat.com/browse/SRVKP-1687
+
+* The Horizontal Pod Autoscaler can manage the replica count of deployments controlled by the {pipelines-title} Operator. From this release onward, the {pipelines-title} Operator will not reset the replica count of deployments managed by it, if the count is changed by an end user or an on cluster agent. However, the replicas will be reset when you upgrade the {pipelines-title} Operator.
```

Comment:
Suggest: 'on-cluster agent' to increase clarity.
Suggest: From this release onward, if the count is changed by an end user or an on cluster agent, the {pipelines-title} Operator will not reset the replica count of deployments managed by it. 
That way an user cna simply skim the second part if it is not relevant for them.